pages/portfolio.html

```diff
@@ -1,12 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="../stylesheets/style.css" rel="stylesheet" />
+	<link href="../stylesheets/portfolio.css" rel="stylesheet" />
+	<title>About Me</title>
+	<!--Google font link-->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+	<!--Adobe font link--> 
+	<link rel="stylesheet" href="https://use.typekit.net/yxs0wfp.css">
 </head>
+
 <body>
-  
-</body>
-</html>
\ No newline at end of file
+<header>
+    <nav>
+        <a href="../index.html"><img src="../assets/adriana-gutierrez-logo-white.png" alt="logo" id="header-logo"></a>
+        <a href="../index.html" class="nav-link">Home</a>
+        <a href="about.html" class="nav-link">About</a>
+        <a href="portfolio.html" class="nav-link">Portfolio</a>
+    </nav>
+</header>
+
+	
+    <section class="container">
+        <div id="lewis-structures" class="project">
+            <a href="google.com">
+                <h1>Lewis Structures</h1>
+            </a>
+            <div class="tags">
+                <p>python</p>
+                <p>REST API</p>
+                <p>flask</p>
+                <p>relational-databases</p>
+                <p>postgreSQL</p>
+                <p>TDD</p>
+            </div>
```

Comment:
Since this is a list of tags, I would suggest using an `ul` element and CSS styling as needed. When the tags better represent their content, it helps understanding code for long term maintenance and can give users with screen readers more clues about the content and intended structure of the page.